Building docker image keeps failing

Trying to compile the docker image locally with `docker compose build` but it keeps failing both on macOS M2 & Linux x86_64 with the following output

```
 > [osmx builder 8/8] RUN make -j$(nproc):
34.35 make[1]: *** Waiting for unfinished jobs....
52.52 3 warnings generated.
53.24 3 warnings generated.
56.15 [100%] Linking CXX executable osmx
56.75 [100%] Built target osmx
62.08 3 warnings generated.
71.72 3 warnings generated.
71.76 [100%] Linking CXX static library libosmx-static.a
71.87 [100%] Built target osmx-static
71.87 make: *** [Makefile:163: all] Error 2
------
failed to solve: process "/bin/sh -c make -j$(nproc)" did not complete successfully: exit code: 2
```

~Any plans for a precompiled image? 👀~

My bad I see it's available here: https://github.com/ellenhp/airmail/pkgs/container/airmail

```> [osmx builder 8/8] RUN make -j$(nproc):
0.481 Scanning dependencies of target build_lmdb
0.484 make[3]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
0.488 Scanning dependencies of target roaring
0.490 Scanning dependencies of target kj
0.499 [  2%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/array_util.c.o
0.499 [  2%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/bitset_util.c.o
0.499 [  6%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/array.c++.o
0.499 [  6%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/common.c++.o
0.499 [  6%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/containers/mixed_intersection.c.o
0.499 [  6%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/containers/array.c.o
0.499 [  6%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/containers/containers.c.o
0.499 [  6%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/containers/convert.c.o
0.499 [  6%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/containers/bitset.c.o
0.499 [  6%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/containers/mixed_union.c.o
0.499 [  8%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/containers/mixed_subset.c.o
0.499 [  8%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/containers/mixed_equal.c.o
0.500 [  8%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/exception.c++.o
0.500 [ 10%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/io.c++.o
0.500 [ 10%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/debug.c++.o
0.500 [ 10%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/memory.c++.o
0.501 [ 10%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/containers/mixed_negation.c.o
0.501 [ 12%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/containers/mixed_andnot.c.o
0.501 [ 12%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/string.c++.o
0.501 [ 12%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/containers/run.c.o
0.501 [ 12%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/mutex.c++.o
0.502 [ 12%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/table.c++.o
0.502 [ 12%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/thread.c++.o
0.502 [ 12%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/roaring.c.o
0.502 [ 12%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/containers/mixed_xor.c.o
0.502 [ 12%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/roaring_priority_queue.c.o
0.512 [ 14%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/main.c++.o
0.512 [ 16%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/hash.c++.o
0.514 [ 16%] Building C object vendor/CRoaring/src/CMakeFiles/roaring.dir/roaring_array.c.o
0.514 [ 18%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/arena.c++.o
0.554 [ 18%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/test-helpers.c++.o
0.587 Scanning dependencies of target s2
0.597 [ 18%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/units.c++.o
0.612 [ 18%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/base/stringprintf.cc.o
0.639 [ 20%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/encoding.c++.o
0.640 [ 20%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/refcount.c++.o
0.640 [ 20%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/base/strtoint.cc.o
0.702 [ 20%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/string-tree.c++.o
0.703 [ 20%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/time.c++.o
0.737 /usr/src/OSMExpress/vendor/CRoaring/src/array_util.c:640:24: warning: comparison of integers of different signs: 'int32_t' (aka 'int') and 'size_t' (aka 'unsigned long') [-Wsign-compare]
0.737           assert(count <= i_a);
0.737                  ~~~~~ ^  ~~~
0.737 /usr/include/assert.h:98:7: note: expanded from macro 'assert'
0.737     ((expr)                                                             \
0.737       ^~~~
0.737 /usr/src/OSMExpress/vendor/CRoaring/src/array_util.c:641:20: warning: comparison of integers of different signs: 'int32_t' (aka 'int') and 'size_t' (aka 'unsigned long') [-Wsign-compare]
0.737           if(count < i_a) {
0.737              ~~~~~ ^ ~~~
0.776 [ 22%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/filesystem.c++.o
0.798 [ 22%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/filesystem-disk-unix.c++.o
0.807 [ 22%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/filesystem-disk-win32.c++.o
0.807 [ 22%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/encoded_s2cell_id_vector.cc.o
0.810 [ 22%] Building CXX object vendor/capnproto/c++/src/kj/CMakeFiles/kj.dir/parse/char.c++.o
0.814 [ 24%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/encoded_s2point_vector.cc.o
0.827 [ 24%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/encoded_s2shape_index.cc.o
0.832 [ 24%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/encoded_string_vector.cc.o
0.843 [ 24%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/id_set_lexicon.cc.o
0.850 [ 24%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/r2rect.cc.o
0.867 [ 28%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s1interval.cc.o
0.867 [ 28%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/mutable_s2shape_index.cc.o
0.877 [ 28%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s1angle.cc.o
0.887 [ 28%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s1chord_angle.cc.o
0.887 [ 28%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2boolean_operation.cc.o
0.907 [ 28%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2builder.cc.o
0.917 [ 30%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2builder_graph.cc.o
0.917 [ 30%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2builderutil_find_polygon_degeneracies.cc.o
0.922 [ 30%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2builderutil_lax_polygon_layer.cc.o
0.927 [ 30%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2builderutil_closed_set_normalizer.cc.o
0.987 [ 32%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2builderutil_s2point_vector_layer.cc.o
0.987 [ 32%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2builderutil_s2polygon_layer.cc.o
0.996 [ 32%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2builderutil_s2polyline_layer.cc.o
0.997 2 warnings generated.
1.010 [ 34%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2builderutil_snap_functions.cc.o
1.027 [ 34%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2builderutil_s2polyline_vector_layer.cc.o
1.057 [ 34%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2cap.cc.o
1.097 [ 34%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2cell.cc.o
1.209 [ 34%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2cell_id.cc.o
1.217 [ 36%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2cell_index.cc.o
1.227 [ 36%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2cell_union.cc.o
1.267 [ 36%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2centroids.cc.o
1.325 [ 38%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2closest_cell_query.cc.o
1.357 [ 38%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2closest_edge_query.cc.o
1.680 [ 38%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2closest_point_query.cc.o
1.686 [ 38%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2contains_vertex_query.cc.o
1.752 [ 40%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2convex_hull_query.cc.o
1.856 [ 40%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2coords.cc.o
1.995 [ 40%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2crossing_edge_query.cc.o
2.080 [ 40%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2debug.cc.o
2.265 [ 42%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2edge_clipping.cc.o
2.265 [ 42%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2earth.cc.o
2.327 [ 42%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2edge_crosser.cc.o
2.357 [ 44%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2edge_crossings.cc.o
2.357 [ 44%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2edge_distances.cc.o
2.477 [ 44%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2edge_tessellator.cc.o
2.491 [ 44%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2latlng.cc.o
2.496 [ 44%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2error.cc.o
2.512 [ 46%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2furthest_edge_query.cc.o
2.523 ar: creating liblmdb.a
2.528 [ 46%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2latlng_rect.cc.o
2.541 [ 46%] Built target build_lmdb
2.572 [ 46%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2latlng_rect_bounder.cc.o
2.576 [ 48%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2lax_loop_shape.cc.o
2.609 [ 48%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2lax_polygon_shape.cc.o
2.626 [ 48%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2lax_polyline_shape.cc.o
2.668 [ 48%] Linking C static library libroaring.a
2.697 [ 48%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2loop.cc.o
2.708 [ 51%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2loop_measures.cc.o
2.737 [ 51%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2measures.cc.o
2.747 [ 51%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2metrics.cc.o
2.786 [ 51%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2min_distance_targets.cc.o
2.807 [ 53%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2max_distance_targets.cc.o
2.820 [ 53%] Built target roaring
2.867 [ 53%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2padded_cell.cc.o
2.884 [ 55%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2point_region.cc.o
2.887 [ 55%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2point_compression.cc.o
2.897 [ 55%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2pointutil.cc.o
2.938 [ 55%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2polygon.cc.o
3.067 [ 55%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2polyline.cc.o
3.087 [ 55%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2polyline_measures.cc.o
3.093 [ 55%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2predicates.cc.o
3.093 [ 55%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2polyline_simplifier.cc.o
3.107 [ 57%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2polyline_alignment.cc.o
3.127 [ 57%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2r2rect.cc.o
3.136 [ 59%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2projections.cc.o
3.172 [ 59%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2region.cc.o
3.291 [ 61%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2region_term_indexer.cc.o
3.367 [ 61%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2region_coverer.cc.o
3.417 [ 63%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2shape_index.cc.o
3.419 [ 63%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2region_intersection.cc.o
3.437 [ 63%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2region_union.cc.o
3.451 [ 63%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2shape_index_buffered_region.cc.o
3.513 [ 63%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2shape_index_measures.cc.o
3.548 [ 63%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2shape_measures.cc.o
3.549 [ 65%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2shapeutil_build_polygon_boundaries.cc.o
3.570 [ 65%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2shapeutil_coding.cc.o
3.617 [ 65%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2shapeutil_contains_brute_force.cc.o
3.699 [ 69%] Linking CXX static library libkj.a
3.699 [ 69%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2shapeutil_edge_iterator.cc.o
3.700 [ 69%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2shapeutil_get_reference_point.cc.o
3.717 [ 69%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2shapeutil_range_iterator.cc.o
3.797 [ 69%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2shapeutil_visit_crossing_edge_pairs.cc.o
3.817 [ 69%] Built target kj
3.877 Scanning dependencies of target capnp
3.895 [ 71%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2text_format.cc.o
3.903 [ 71%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/c++.capnp.c++.o
3.946 [ 71%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/s2wedge_relations.cc.o
3.987 [ 71%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/strings/serialize.cc.o
3.997 [ 71%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/strings/ostringstream.cc.o
4.007 [ 73%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/third_party/absl/base/dynamic_annotations.cc.o
4.028 [ 73%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/blob.c++.o
4.037 [ 73%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/third_party/absl/base/internal/raw_logging.cc.o
4.044 [ 75%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/arena.c++.o
4.087 [ 75%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/third_party/absl/base/internal/throw_delegate.cc.o
4.087 [ 75%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/third_party/absl/numeric/int128.cc.o
4.117 [ 75%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/layout.c++.o
4.267 [ 77%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/third_party/absl/strings/ascii.cc.o
4.272 [ 77%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/third_party/absl/strings/match.cc.o
4.303 [ 77%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/list.c++.o
4.303 [ 79%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/third_party/absl/strings/str_cat.cc.o
4.325 [ 79%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/third_party/absl/strings/numbers.cc.o
4.327 [ 79%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/third_party/absl/strings/str_split.cc.o
4.337 [ 79%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/third_party/absl/strings/string_view.cc.o
4.367 [ 79%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/third_party/absl/strings/strip.cc.o
4.405 [ 79%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/any.c++.o
4.437 [ 81%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/third_party/absl/strings/internal/memutil.cc.o
4.467 [ 81%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/util/bits/bit-interleave.cc.o
4.467 [ 81%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/util/bits/bits.cc.o
4.502 [ 81%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/util/coding/coder.cc.o
4.528 [ 83%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/util/coding/varint.cc.o
4.537 [ 83%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/util/math/exactfloat/exactfloat.cc.o
4.553 [ 83%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/util/units/length-units.cc.o
4.567 [ 83%] Building CXX object vendor/s2geometry/CMakeFiles/s2.dir/src/s2/util/math/mathutil.cc.o
4.587 [ 85%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/message.c++.o
4.607 [ 85%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/schema.capnp.c++.o
4.657 [ 85%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/serialize.c++.o
4.677 [ 85%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/serialize-packed.c++.o
4.697 [ 87%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/schema.c++.o
4.717 [ 87%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/schema-loader.c++.o
4.728 [ 87%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/dynamic.c++.o
4.731 [ 87%] Building CXX object vendor/capnproto/c++/src/capnp/CMakeFiles/capnp.dir/stringify.c++.o
7.069 [ 89%] Linking CXX static library libcapnp.a
7.077 [ 91%] Linking CXX static library libs2.a
7.109 [ 91%] Built target capnp
7.148 [ 91%] Built target s2
7.175 Scanning dependencies of target osmxTest
7.185 [ 93%] Building CXX object CMakeFiles/osmxTest.dir/test/test_region.cpp.o
7.185 [ 93%] Building CXX object CMakeFiles/osmxTest.dir/src/region.cpp.o
7.200 Scanning dependencies of target osmx
7.200 Scanning dependencies of target osmx-static
7.201 /usr/src/OSMExpress/test/test_region.cpp:2:10: fatal error: 'catch2/catch.hpp' file not found
7.201 #include "catch2/catch.hpp"
7.201          ^~~~~~~~~~~~~~~~~~
7.213 [ 93%] Building CXX object CMakeFiles/osmx.dir/src/cmd.cpp.o
7.213 [ 93%] Building CXX object CMakeFiles/osmx-static.dir/src/storage.cpp.o
7.213 [ 93%] Building CXX object CMakeFiles/osmx.dir/src/storage.cpp.o
7.214 [ 93%] Building CXX object CMakeFiles/osmx-static.dir/src/expand.cpp.o
7.214 [ 93%] Building CXX object CMakeFiles/osmx-static.dir/src/extract.cpp.o
7.214 [ 95%] Building CXX object CMakeFiles/osmx-static.dir/src/update.cpp.o
7.215 [ 95%] Building CXX object CMakeFiles/osmx.dir/src/expand.cpp.o
7.215 [ 95%] Building CXX object CMakeFiles/osmx-static.dir/src/region.cpp.o
7.215 [ 95%] Building CXX object CMakeFiles/osmx.dir/src/update.cpp.o
7.216 [ 95%] Building CXX object CMakeFiles/osmx.dir/src/region.cpp.o
7.217 [ 97%] Building CXX object CMakeFiles/osmx.dir/src/extract.cpp.o
7.811 1 error generated.
7.819 make[2]: *** [CMakeFiles/osmxTest.dir/build.make:63: CMakeFiles/osmxTest.dir/test/test_region.cpp.o] Error 1
7.819 make[2]: *** Waiting for unfinished jobs....
8.720 /usr/src/OSMExpress/src/update.cpp:53:31: warning: 'operator long' is deprecated [-Wdeprecated-declarations]
8.720         metadata.setTimestamp(node.timestamp());
8.720                               ^
8.720 /usr/src/OSMExpress/vendor/libosmium/include/osmium/osm/timestamp.hpp:253:9: note: 'operator long' has been explicitly marked deprecated here
8.720         OSMIUM_DEPRECATED constexpr operator time_t() const noexcept { // NOLINT(google-explicit-constructor, hicpp-explicit-conversions)
8.720         ^
8.720 /usr/src/OSMExpress/vendor/libosmium/include/osmium/util/compatibility.hpp:47:43: note: expanded from macro 'OSMIUM_DEPRECATED'
8.720 # define OSMIUM_DEPRECATED __attribute__((deprecated))
8.720                                           ^
8.724 /usr/src/OSMExpress/src/update.cpp:107:29: warning: 'operator long' is deprecated [-Wdeprecated-declarations]
8.724       metadata.setTimestamp(way.timestamp());
8.724                             ^
8.724 /usr/src/OSMExpress/vendor/libosmium/include/osmium/osm/timestamp.hpp:253:9: note: 'operator long' has been explicitly marked deprecated here
8.724         OSMIUM_DEPRECATED constexpr operator time_t() const noexcept { // NOLINT(google-explicit-constructor, hicpp-explicit-conversions)
8.724         ^
8.724 /usr/src/OSMExpress/vendor/libosmium/include/osmium/util/compatibility.hpp:47:43: note: expanded from macro 'OSMIUM_DEPRECATED'
8.724 # define OSMIUM_DEPRECATED __attribute__((deprecated))
8.724                                           ^
8.727 /usr/src/OSMExpress/src/update.cpp:180:29: warning: 'operator long' is deprecated [-Wdeprecated-declarations]
8.727       metadata.setTimestamp(relation.timestamp());
8.727                             ^
8.727 /usr/src/OSMExpress/vendor/libosmium/include/osmium/osm/timestamp.hpp:253:9: note: 'operator long' has been explicitly marked deprecated here
8.727         OSMIUM_DEPRECATED constexpr operator time_t() const noexcept { // NOLINT(google-explicit-constructor, hicpp-explicit-conversions)
8.727         ^
8.727 /usr/src/OSMExpress/vendor/libosmium/include/osmium/util/compatibility.hpp:47:43: note: expanded from macro 'OSMIUM_DEPRECATED'
8.727 # define OSMIUM_DEPRECATED __attribute__((deprecated))
8.727                                           ^
8.728 /usr/src/OSMExpress/src/expand.cpp:162:29: warning: 'operator long' is deprecated [-Wdeprecated-declarations]
8.728       metadata.setTimestamp(node.timestamp());
8.728                             ^
8.728 /usr/src/OSMExpress/vendor/libosmium/include/osmium/osm/timestamp.hpp:253:9: note: 'operator long' has been explicitly marked deprecated here
8.728         OSMIUM_DEPRECATED constexpr operator time_t() const noexcept { // NOLINT(google-explicit-constructor, hicpp-explicit-conversions)
8.728         ^
8.728 /usr/src/OSMExpress/vendor/libosmium/include/osmium/util/compatibility.hpp:47:43: note: expanded from macro 'OSMIUM_DEPRECATED'
8.728 # define OSMIUM_DEPRECATED __attribute__((deprecated))
8.728                                           ^
8.728 /usr/src/OSMExpress/src/expand.cpp:185:27: warning: 'operator long' is deprecated [-Wdeprecated-declarations]
8.728     metadata.setTimestamp(way.timestamp());
8.728                           ^
8.728 /usr/src/OSMExpress/vendor/libosmium/include/osmium/osm/timestamp.hpp:253:9: note: 'operator long' has been explicitly marked deprecated here
8.728         OSMIUM_DEPRECATED constexpr operator time_t() const noexcept { // NOLINT(google-explicit-constructor, hicpp-explicit-conversions)
8.728         ^
8.728 /usr/src/OSMExpress/vendor/libosmium/include/osmium/util/compatibility.hpp:47:43: note: expanded from macro 'OSMIUM_DEPRECATED'
8.728 # define OSMIUM_DEPRECATED __attribute__((deprecated))
8.728                                           ^
8.729 /usr/src/OSMExpress/src/expand.cpp:219:27: warning: 'operator long' is deprecated [-Wdeprecated-declarations]
8.729     metadata.setTimestamp(relation.timestamp());
8.729                           ^
8.729 /usr/src/OSMExpress/vendor/libosmium/include/osmium/osm/timestamp.hpp:253:9: note: 'operator long' has been explicitly marked deprecated here
8.729         OSMIUM_DEPRECATED constexpr operator time_t() const noexcept { // NOLINT(google-explicit-constructor, hicpp-explicit-conversions)
8.729         ^
8.729 /usr/src/OSMExpress/vendor/libosmium/include/osmium/util/compatibility.hpp:47:43: note: expanded from macro 'OSMIUM_DEPRECATED'
8.729 # define OSMIUM_DEPRECATED __attribute__((deprecated))
8.729                                           ^
8.732 /usr/src/OSMExpress/src/expand.cpp:162:29: warning: 'operator long' is deprecated [-Wdeprecated-declarations]
8.732       metadata.setTimestamp(node.timestamp());
8.732                             ^
8.732 /usr/src/OSMExpress/vendor/libosmium/include/osmium/osm/timestamp.hpp:253:9: note: 'operator long' has been explicitly marked deprecated here
8.733         OSMIUM_DEPRECATED constexpr operator time_t() const noexcept { // NOLINT(google-explicit-constructor, hicpp-explicit-conversions)
8.733         ^
8.733 /usr/src/OSMExpress/vendor/libosmium/include/osmium/util/compatibility.hpp:47:43: note: expanded from macro 'OSMIUM_DEPRECATED'
8.733 # define OSMIUM_DEPRECATED __attribute__((deprecated))
8.733                                           ^
8.733 /usr/src/OSMExpress/src/expand.cpp:185:27: warning: 'operator long' is deprecated [-Wdeprecated-declarations]
8.733     metadata.setTimestamp(way.timestamp());
8.733                           ^
8.733 /usr/src/OSMExpress/vendor/libosmium/include/osmium/osm/timestamp.hpp:253:9: note: 'operator long' has been explicitly marked deprecated here
8.733         OSMIUM_DEPRECATED constexpr operator time_t() const noexcept { // NOLINT(google-explicit-constructor, hicpp-explicit-conversions)
8.733         ^
8.733 /usr/src/OSMExpress/vendor/libosmium/include/osmium/util/compatibility.hpp:47:43: note: expanded from macro 'OSMIUM_DEPRECATED'
8.733 # define OSMIUM_DEPRECATED __attribute__((deprecated))
8.733                                           ^
8.735 /usr/src/OSMExpress/src/expand.cpp:219:27: warning: 'operator long' is deprecated [-Wdeprecated-declarations]
8.735     metadata.setTimestamp(relation.timestamp());
8.735                           ^
8.735 /usr/src/OSMExpress/vendor/libosmium/include/osmium/osm/timestamp.hpp:253:9: note: 'operator long' has been explicitly marked deprecated here
8.735         OSMIUM_DEPRECATED constexpr operator time_t() const noexcept { // NOLINT(google-explicit-constructor, hicpp-explicit-conversions)
8.735         ^
8.735 /usr/src/OSMExpress/vendor/libosmium/include/osmium/util/compatibility.hpp:47:43: note: expanded from macro 'OSMIUM_DEPRECATED'
8.735 # define OSMIUM_DEPRECATED __attribute__((deprecated))
8.735                                           ^
8.739 /usr/src/OSMExpress/src/update.cpp:53:31: warning: 'operator long' is deprecated [-Wdeprecated-declarations]
8.739         metadata.setTimestamp(node.timestamp());
8.739                               ^
8.739 /usr/src/OSMExpress/vendor/libosmium/include/osmium/osm/timestamp.hpp:253:9: note: 'operator long' has been explicitly marked deprecated here
8.739         OSMIUM_DEPRECATED constexpr operator time_t() const noexcept { // NOLINT(google-explicit-constructor, hicpp-explicit-conversions)
8.739         ^
8.739 /usr/src/OSMExpress/vendor/libosmium/include/osmium/util/compatibility.hpp:47:43: note: expanded from macro 'OSMIUM_DEPRECATED'
8.739 # define OSMIUM_DEPRECATED __attribute__((deprecated))
8.739                                           ^
8.745 /usr/src/OSMExpress/src/update.cpp:107:29: warning: 'operator long' is deprecated [-Wdeprecated-declarations]
8.745       metadata.setTimestamp(way.timestamp());
8.745                             ^
8.745 /usr/src/OSMExpress/vendor/libosmium/include/osmium/osm/timestamp.hpp:253:9: note: 'operator long' has been explicitly marked deprecated here
8.745         OSMIUM_DEPRECATED constexpr operator time_t() const noexcept { // NOLINT(google-explicit-constructor, hicpp-explicit-conversions)
8.745         ^
8.745 /usr/src/OSMExpress/vendor/libosmium/include/osmium/util/compatibility.hpp:47:43: note: expanded from macro 'OSMIUM_DEPRECATED'
8.745 # define OSMIUM_DEPRECATED __attribute__((deprecated))
8.745                                           ^
8.748 /usr/src/OSMExpress/src/update.cpp:180:29: warning: 'operator long' is deprecated [-Wdeprecated-declarations]
8.748       metadata.setTimestamp(relation.timestamp());
8.748                             ^
8.748 /usr/src/OSMExpress/vendor/libosmium/include/osmium/osm/timestamp.hpp:253:9: note: 'operator long' has been explicitly marked deprecated here
8.748         OSMIUM_DEPRECATED constexpr operator time_t() const noexcept { // NOLINT(google-explicit-constructor, hicpp-explicit-conversions)
8.748         ^
8.748 /usr/src/OSMExpress/vendor/libosmium/include/osmium/util/compatibility.hpp:47:43: note: expanded from macro 'OSMIUM_DEPRECATED'
8.748 # define OSMIUM_DEPRECATED __attribute__((deprecated))
8.748                                           ^
11.54 make[1]: *** [CMakeFiles/Makefile2:476: CMakeFiles/osmxTest.dir/all] Error 2
11.54 make[1]: *** Waiting for unfinished jobs....
23.19 3 warnings generated.
23.39 3 warnings generated.
23.40 3 warnings generated.
23.76 3 warnings generated.
27.42 [ 97%] Linking CXX static library libosmx-static.a
27.53 [ 97%] Built target osmx-static
27.69 [100%] Linking CXX executable osmx
28.11 [100%] Built target osmx
28.11 make: *** [Makefile:163: all] Error 2
------
failed to solve: process "/bin/sh -c make -j$(nproc)" did not complete successfully: exit code: 2```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM rust:1.76 as build
+FROM rust:1.81 as build
 
 RUN apt update && apt install -y libssl-dev clang pkg-config
 

--- a/airmail_indexer/Dockerfile.osmx
+++ b/airmail_indexer/Dockerfile.osmx
@@ -14,7 +14,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install 
 WORKDIR /usr/src/
 RUN git clone --recurse-submodules https://github.com/protomaps/OSMExpress.git
 WORKDIR /usr/src/OSMExpress
-RUN git checkout 24cdc6cca55f45b4019e2d490c2321c366ca3362
+RUN git checkout 331020a9e027c42c29baa423adeb726332ca2105
 
 
 RUN cmake -DCMAKE_BUILD_TYPE=Release .

